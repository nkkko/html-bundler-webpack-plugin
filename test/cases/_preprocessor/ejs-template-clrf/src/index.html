<!-- IMPORTANT: this file MUST be in windows format with CRLF new line separators -->
<!DOCTYPE html>
<html>
  <head>
    <title><%= title %></title>
		<!-- IMPORTANT: test TAB in HTML -->
		<script src="./main.js" defer="defer"></script>
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
</html>