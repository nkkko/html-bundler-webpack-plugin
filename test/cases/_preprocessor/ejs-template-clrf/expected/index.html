<!-- IMPORTANT: this file MUST be in windows format with CRLF new line separators -->
<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
		<!-- IMPORTANT: test TAB in HTML -->
		<script src="assets/js/main.5317c1f6.js" defer="defer"></script>
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
</html>